frontend/institution/management_members.html

```diff
@@ -1,5 +1,5 @@
 <!-- CONTENT -->
-<div flex-gt-lg="80" flex-lg="90" layout-align="center" layout-fill>
+<div layout-padding flex-gt-sm="80" flex-gt-lg="80" flex-lg="90" layout-align="center" layout-fill>
```

Comment:
The property flex it's not correct.  It's missing the flex default, and the tag "-gt-" it's only used in the last definition of the div:

`<div flex flex-sm="80" flex-md="80" flex-lg="70" flex-gt-lg="60" ...>`